frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -145,7 +145,7 @@ export default function AppLayout() {
   function MainNavSecondary() {
     const { hasChannelManagement } = useInfo();
     return (
-      <nav className="grid items-start p-2 text-sm font-medium lg:px-4">
+      <nav className="grid items-start px-4 py-2 text-sm font-medium">
```

Comment:
@reneaaron This adds double the horizontal padding on mobile and I'm worried it will cause issues, was this intentional?

Reply:
On mobile we have the drawer anyway, right? This just removes smaller paddings on `md` screens...

Reply:
Ah true, thanks!